.github/workflows/codeql-query.yml

```diff
@@ -66,13 +66,17 @@ jobs:
         shell: node {0}
         run: |
           const fs = require('fs');
+          const process = require('process');
           const allRepos = JSON.parse(fs.readFileSync("instructions.json")).repositories;
           const repoNwos = new Set(${{ toJSON(matrix.repoNwos) }});
           const repositories = allRepos.filter(r => repoNwos.has(r.nwo));
           if ("${{ secrets.TEST_PAT }}") {
             allRepos.forEach(r => r.pat = "${{ secrets.TEST_PAT }}")
           }
-          console.log(`::set-output name=repositories::${JSON.stringify(repositories)}`);
+          const githubOutput = process.env.GITHUB_OUTPUT;
+          const outputFile = fs.openSync(githubOutput, 'a');
+          fs.writeSync(outputFile, `repositories=${JSON.stringify(repositories)}`);
+          fs.closeSync(outputFile);
```

Comment:
`@simon-engledew` helped me with this since this was a bit more complicated than the `>> $GITHUB_OUTPUT` version 😅 

We verified that this step succeeded in https://github.com/github/codeql-variant-analysis-action/actions/runs/3328681685/jobs/5504956701#step:5:111

![image](https://user-images.githubusercontent.com/42641846/198019495-bf23c24e-0b18-4e84-8d9b-56bc345b9207.png)